src/components/import.ts

```diff
@@ -3,6 +3,7 @@ export async function dynamicImport(
   module: string | undefined,
   url: string | undefined
 ) {
+  // @ts-ignore
```

Comment:
was this causing issues or did you forget to remove it?

Reply:
It was causing problems 😕:

```
src/components/import.ts:6:9 - error TS2304: Cannot find name '__webpack_init_sharing__'.

6   await __webpack_init_sharing__('default');
          ~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.
```